test/browser/features/support/skip.rb

```diff
@@ -16,4 +16,12 @@
 
 Before("@skip_chrome_61") do
   skip_this_scenario("Skipping scenario: Not supported") if Maze.config.browser == "chrome_61"
+end
+
+Before("@skip_firefox_60") do
```

Comment:
Is it worth generating these so we don't need to keep adding specific versions here? e.g. something like:

```ruby
["chrome", "firefox", "safari", "edge"].each do |browser|
  1.upto(1_000) do |version|
    Before("@skip_#{browser}_#{version}") do
      skip_this_scenario("Skipping scenario: Not supported") if Maze.config.browser == "#{browser}_#{version}"
    end
  end
end
```